account-integrations/safe/test/utils/TestHelper.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "forge-std/Test.sol";
+import {EntryPoint, UserOperation} from "account-abstraction/contracts/core/EntryPoint.sol";
+import {Webauthn} from "../../src/WebAuthn.sol";
+
+/* solhint-disable private-vars-leading-underscore */
+
+abstract contract TestHelper is Test {
+    EntryPoint public entryPoint;
+    address internal entryPointAddress;
+
+    address internal constant ALICE = address(1);
+    address internal constant BOB = address(2);
+    address[] internal testAccounts = [ALICE, BOB];
+
+    constructor() {
+        entryPoint = new EntryPoint();
+        entryPointAddress = address(entryPoint);
+    }
+
+    function buildUserOp() public pure returns (UserOperation memory userOp) {
+        address sender = ALICE;
+        uint256 nonce = 0;
+        bytes memory initCode = hex"00";
+        bytes memory callData = hex"00";
+        uint256 callGasLimit = 0;
+        uint256 verificationGasLimit = 0;
+        uint256 preVerificationGas = 0;
+        uint256 maxFeePerGas = 0;
+        uint256 maxPriorityFeePerGas = 0;
+        bytes memory paymasterAndData = hex"00";
+        bytes memory signature = hex"00";
+
+        userOp = UserOperation(
+            sender,
+            nonce,
+            initCode,
+            callData,
+            callGasLimit,
+            verificationGasLimit,
+            preVerificationGas,
+            maxFeePerGas,
+            maxPriorityFeePerGas,
+            paymasterAndData,
+            signature
+        );
+    }
+
+    function getPublicKey()
```

Comment:
Same here, if this is a WebAuthN public key it may be worth reflecting that in the name, since we will eventually have different types of public keys

Reply:
Agree, updated in follow up